src/database.rst

```diff
@@ -209,8 +209,6 @@ following way:
             $this->assertSame(2, 1 + 1);
         }
     }
-    ?>
-
```

Comment:
empty line between code and text shall be restored

Reply:
Fixed, thanks for noticing this :+1: